contract/contracts/Subscriptions.sol

```diff
@@ -219,14 +220,17 @@ contract Subscriptions {
             delete _subscriptions[user];
         }
 
+        bool success = token.transfer(user, unlocked(sub));
+        require(success);
+
         emit Unsubscribe(user);
     }
 
     // Extend a user's subscription.
     function extend(address user, uint64 end) public {
         require(user != address(0), 'user is null');
         uint64 currentBlock = uint64(block.number);
-        Subscription storage sub = _subscriptions[user];
+        Subscription memory sub = _subscriptions[user];
```

Comment:
Same as above, I think this is more expensive than using storage.

Reply:
[240afab](https://github.com/edgeandnode/subscription-payments-contract/pull/9/commits/240afabdb803f9397b372df605cf24d6360fce2b)